src/providers/google/calendar-event.ts

```diff
@@ -175,23 +175,23 @@ export default class CalendarEventHandler extends GoogleHandler {
       let eventHistory: SchemaEvent[] = [];
 
       // Iterate over each calendar
-      for (const calendar of calendarList) {
+      for (let i = 0; i < calendarList.length; i++) {
```

Comment:
Moving forward, please avoid using this type of for statement and use the `for (const calendar of calendarList)` format.